src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class Storage {
+    private File file;
+
+    /**
+     * Creates necessary directories to the file path if not present
+     *
+     * @param filePath
+     * @return TaskList
+     * @throws DukeException
```

Comment:
Perhaps you could describe the circumstances which would cause DukeException to be thrown?